test/run.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-dir="$(dirname "$(realpath "$BASH_SOURCE")")"
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
```

Comment:
is it works on mac os, after change?

Reply:
yes, if you exported path '/usr/local/opt/coreutils/libexec/gnubin' to gnubin  :)